src/modules/devops-guide/pages/server-plugins/index.adoc

```diff
@@ -0,0 +1,41 @@
+include::ROOT:partial$variables.adoc[]
+
+= Server plugins
```

Comment:
Good base; refer to http://localhost:8080/corteza-docs/2021.9/devops-guide/maintenance/i18n.html for an idea of how I would like this to look like.

The sections should tell a story where each bit tells everything it should (section B should not append anything that section A should state; it can, however, reference to it).